packages/wallet-core/src/serde/wire-format/example.ts

```diff
@@ -45,10 +45,9 @@ export const wireStateFormat: WireState = {
   ],
   turnNum: 1,
   signatures: [
-    '0xef7e226a43c52d4b8f7b14f13acdf9e75d871ea5c51235fbc4d538acf84c61c4727431f0cc83d0f566e222a21d35ae4d8d2a0dd4428cba7bf95bf7b3f11ad0c61c'
+    '0x59069c99dd52a7766e439ef9344e5cee7a51625ec366095cbd7bad66075a54e924ab48b08a683deff7e3642d8017178e55990b2073d169eec64a7fef60648a6a1b'
```

Comment:
I had to regenerate this signature manually, which was a pain.

Reply:
It's a nice touch to leave [instructions in a comment](https://github.com/statechannels/the-graph/blob/master/packages/statechannels-contracts/test/attestationApp.test.ts#L67-L74) on how to generate the signature. Thank @lalexgap for that!